config/locales/en.yml

```diff
@@ -3,6 +3,7 @@ en:
     DATE_FORMATS: &DATE_FORMATS
       default: '%-d %b %Y'  # 3 Jan 2019
       compact: '%d/%m/%Y'   # 03/01/2019
+      long: '%a %-d %b %y'   # Fri 1 Sep 2023
```

Comment:
Not important but i would've preferred it to use the same dates as lines 4 and 5 so `Thur 3 Jan 2019`